orchestrator-salt/src/main/resources/salt/salt/top.sls

```diff
@@ -28,6 +28,7 @@ base:
 
   'G@roles:manager_server':
     - postgresql
+    - postgresql.disaster_recovery
```

Comment:
I don't think we need to include it here. Whenever we invoke restore or backup it will include the init.sls and create the scripts.

Reply:
Ok, this `top` file controls what is _initially_ moved to a minion then?
We'll still receive the states when we use Java to run the `state.apply` function, so we don't have to include it here?

Reply:
This top file describes what will run during highstate. We call highstate very frequently to make sure everything is in place. Backup and restore will be called occasionally so we don't have to include in the top file, it will just create the files so it's not much, but it adds some time to the execution.